ega-data-api-res/src/main/java/eu/elixir/ega/ebi/reencryptionmvc/ReEncryptionMvcApplication.java

```diff
@@ -16,13 +16,15 @@
 package eu.elixir.ega.ebi.reencryptionmvc;
 
 import org.springframework.boot.SpringApplication;
-import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.cache.annotation.EnableCaching;
 import org.springframework.cloud.netflix.eureka.EnableEurekaClient;
 import org.springframework.cloud.netflix.hystrix.EnableHystrix;
 import org.springframework.cloud.netflix.hystrix.dashboard.EnableHystrixDashboard;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.Configuration;
 
-@SpringBootApplication
```

Comment:
The reason is because adding the spring db dependency makes it so that if you start the program without database configuration it fails.

 So if you don't need it you either
* change the autoconfiguration: (that is what I did)

* modify the properties configuration to add:
> spring.autoconfigure.exclude=org.springframework.boot.autoconfigure.orm.jpa.HibernateJpaAutoConfiguration,\
    org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration,\
    org.springframework.boot.autoconfigure.jdbc.DataSourceTransactionManagerAutoConfiguration,\
    org.springframework.boot.autoconfigure.data.web.SpringDataWebAutoConfiguration

 so the app does not try to autoconfigure any database

I can change the code and use again @SpringBootApplication but then the properties should be added if you don't need a db